### ISSUE TITLE
Routing (Navigation)

### ISSUE THREAD

**Issue by darkscript** [KALRO-Kenya/kalro-maps#3]
A context menu on left drawer list should have option "Directions to this Center"

**Comment by jometho** [comment]
Yes, that would be great. With this, the locations (lat,long) of the center are used as the End Point, and the user can enter their start point or we pre-feed the users location if available as the start point.
